htdocs/ecommerceng/core/triggers/interface_90_modECommerceng_ECommerceng.class.php

```diff
@@ -801,8 +805,72 @@ function run_trigger($action,$object,$user,$langs,$conf)
                 return 1;
             }            
         }
-        
-		return 0;
+
+        if ($action == 'ECOMMERCE_PRICE_LEVEL_MODIFY')
```

Comment:
Why did your put this part of code into trigger and not directly in line 139 of the file
htdocs/ecommerceng/admin/eCommerceSetup.php

The code behind the trigger is ran only if we change a setup in setup page, so why not moving it into setup page ?